src/components/SettingsDialog/ExportBoard/ExportHintHiddenColumns/ExportHintHiddenColumns.scss

```diff
@@ -0,0 +1,48 @@
+@import "src/constants/style.scss";
+
+$icon-text--gap: 20px;
```

Comment:
This is fine, but in case 4px difference do not bother you: `$margin--medium` is 24px. This would reduce the usage of local variables.